pre_commit/resources/hook-tmpl

```diff
@@ -8,6 +8,7 @@ popd > /dev/null
 
```

Comment:
I can't comment quite high enough in this file, but since you're changing the contents you'll need to update the md5 blob at the top and update the list in pre_commit/commands/install_uninstall.py